README.md

```diff
@@ -1,5 +1,5 @@
-About cuda-nvcc-feedstock
-=========================
+About cuda-nvcc-split-feedstock
+===============================
```

Comment:
To fix this, would suggest adding this snippet to the `recipe` and re-rendering again

```YAML
# filename: recipe/meta.yaml
...
extra:
  feedstock-name: cuda-nvcc
  ...
```

Reply:
This is interesting...